lua/gitportal/config.lua

```diff
@@ -15,6 +16,30 @@ M.options = default
 function M.setup(options)
   -- Merge user options with default options
   M.options = vim.tbl_deep_extend("force", {}, default, options or {})
+
+  local commands = {
+    open = function ()
+      require("gitportal").open_file_in_browser()
+    end,
+    link = function ()
+      require("gitportal").open_file_in_neovim()
+    end
+  }
+
+  vim.api.nvim_create_user_command("GitPortal", function (ev)
+    local cmd = commands[ev.fargs[1] or "open"]
```

Comment:
right now I made the default command "open" which opens current file in browser

Reply:
Do we have to do some extra config for this to be accessible to users? Calling it via `:GitPortal` doesn't seem to work for me

Reply:
aah right, with this change you have to call the `setup` function, that is when the user command gets created

Reply:
Ohh I see. That's strange tho, isn't this sufficient? 
https://github.com/trevorhauter/nvtrev2/blob/main/lua/plugins/gitportal.lua#L1

When I change the path to this repo/branch locally, the command still isn't available. Is there something else I should be doing? Or is this perhaps an issue with lazy loading in lazyvim?

Reply:
Hmm, it should work, can you send the config where it doesnt work and you specify this branch?